_posts/2019-06-30-GraphQL-Announcement.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: GraphQL Support for Parse Server
+date: 2019-06-25 22:10 -0000
+comments: true
+author: tomwfox
+categories: [Announcements, Update]
+---
+
+We're glad to announce that Parse Server now has support for GraphQL...
```

Comment:
I think we should work more here... mention the benefits of GraphQL and how this technology is growing and how it can modernize an existing Parse app. We can also mention something like: "Parse Platform is now the easiest way to instant create a GraphQL API"